resources/views/auth/registration_mail.blade.php

```diff
@@ -0,0 +1,63 @@
+@extends('layout.app')
+@section('content')
+<header>
```

Comment:
header should be in the app.blade if same html

Reply:
Understood, sir.